<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signup</class>
 <widget class="QWidget" name="Signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(238, 238, 236);</string>
  </property>
  <widget class="QLabel" name="label_image2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>340</width>
     <height>520</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_qwerty">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 50pt &quot;Waree&quot;;</string>
   </property>
   <property name="text">
    <string>Qwerty</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_login">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Waree&quot;;</string>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_user2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>200</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_signup">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>451</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_email2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>260</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>邮箱</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_passwd2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>320</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_passwd3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>320</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>重复密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_confirm">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>380</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>验证码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>380</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <zorder>lineEdit_user2</zorder>
  <zorder>label_image2</zorder>
  <zorder>label_qwerty</zorder>
  <zorder>label_login</zorder>
  <zorder>button_signup</zorder>
  <zorder>lineEdit_email2</zorder>
  <zorder>lineEdit_passwd2</zorder>
  <zorder>lineEdit_passwd3</zorder>
  <zorder>lineEdit_confirm</zorder>
  <zorder>sendButton</zorder>
 </widget>
 <tabstops>
  <tabstop>lineEdit_user2</tabstop>
  <tabstop>lineEdit_email2</tabstop>
  <tabstop>lineEdit_passwd2</tabstop>
  <tabstop>lineEdit_passwd3</tabstop>
  <tabstop>lineEdit_confirm</tabstop>
  <tabstop>sendButton</tabstop>
  <tabstop>button_signup</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
